src/ReinforcementLearning/SimpleAgent/RandomAgent.h

```diff
@@ -14,26 +15,32 @@ class RandomAgent : public BaseAgent {
     public:
     RandomAgent(){};
     ~RandomAgent(){};
+    std::vector <std::pair<std::vector <bool>, unsigned int>> history;
```

Comment:
This needs an explanatory comment in doxygen style: What are the vector and integer?